scripts/vagrantCloudBasedContainers.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh -ex
```

Comment:
Thanks for the feedback, Tony. I've added some comments to the script and the README.